app/Models/Season.php

```diff
@@ -45,6 +48,28 @@ public static function latestOrId($id)
         return $season;
     }
 
+    public function divisions(): HasMany
+    {
+        return $this->hasMany(Division::class);
+    }
+
+    public function divisionsWithAbsoluteThresholds(): array
+    {
+        $divisions = $this->divisions()->orderBy('threshold')->get();
+        $userCount = $this->userScores()->count();
```

Comment:
I believe this should have `forRanking` filter just like the actual table

also the value is already queried by the pagination thing...